baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/SafeLoggingPropagation.java

```diff
@@ -195,43 +174,87 @@ private Description matchClassOrInterface(ClassTree classTree, ClassSymbol class
         return matchBasedOnToString(classTree, classSymbol, state);
     }
 
+    private static boolean isImmutablesField(
+            ClassSymbol enclosingClass, MethodSymbol methodSymbol, VisitorState state) {
+        return methodSymbol.getModifiers().contains(Modifier.ABSTRACT)
+                || ASTHelpers.hasAnnotation(methodSymbol, "org.immutables.value.Value.Default", state)
```

Comment:
Clarification q: We omit `Redacted` methods here, which means `isGetterMethod` would return false for `test` here: 
```
@Value.Redacted
default String test() { return "test'; }
```

I think we expect `isGetterMethod` to evaluate to true here given we have logic to handle/ignore redacted fields in the branch that evaluates when `isGetterMethod` returns true?

Reply:
It really depends on `immutablesDefaultAsDefault(enclosingClass, state)` below -- by default, `default` methods are considered utility functions, and aren't included as fields in immutables objects. The redacted annotation isn't meaningful on utility methods, and can be ignored.